.github/CODEOWNERS

```diff
@@ -510,7 +510,7 @@ app/swagger/swagger/requests/appeals @department-of-veterans-affairs/backend-rev
 app/swagger/swagger/requests/backend_statuses.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/swagger/swagger/requests/banners.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/swagger/swagger/requests/benefits_reference_data.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/swagger/swagger/requests/burial_claims.rb @department-of-veterans-affairs/mbs-core-team @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+app/swagger/swagger/requests/burial_claims.rb  @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

Comment:
For this line and all of the others where you're deleting `mbs-core-team` and NOT replacing it with your team, can you revert it back to list mbs-core-team? this will help us know which lines still need an updated codeowner.